pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/RealtimeSegmentDataManager.java

```diff
@@ -341,6 +352,13 @@ private void computeKeepIndexing() {
         keepIndexing = false;
       }
     }
+    updateCurrentDocumentCountMetrics();
```

Comment:
This is to periodically update the metrics inside segmentStatusTask, which will be running every 1mins. 
So we will have number of documents metric updated in realtime segment.